src/patternfly/components/Touchspin/touchspin.scss

```diff
@@ -0,0 +1,55 @@
+.pf-c-touchspin {
+  // main
+  --pf-c-touchspin__main--m-active--c-form-control--Color: var(--pf-global--Color--100);
+
+  // units
+  --pf-c-touchspin__main__units--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-touchspin__units__main--MarginLeft: var(--pf-global--spacer--sm);
+
+  // control
+  --pf-c-touchspin__control--m-threshold--c-button--Color: var(--pf-global--disabled-color--100);
+
+  // icon
+  --pf-c-touchspin__control-icon--FontSize: var(--pf-global--FontSize--xs);
+
+  // form control
+  --pf-c-touchspin--c-form-control--Color: var(--pf-global--disabled-color--100);
+  --pf-c-touchspin--c-form-control--MaxWidth: 6ch;
+
+  display: flex;
+  align-items: center;
+
+  .pf-c-form-control {
+    display: inline;
+    max-width: var(--pf-c-touchspin--c-form-control--MaxWidth);
+    color: var(--pf-c-touchspin--c-form-control--Color);
```

Comment:
I don't think you need this. Looking at the designs, the grey text looks to be placeholder text. When the input has a value, it's black text like a regular input.

Reply:
@maryshak1996 could you take a look at this. From the designs my understanding was that the text color is light grey when the touchspin controls arent active / user isnt editing the text. Is the grey to represent placeholder text before the user inputs something or changes the control?